Source/WebInspectorUI/UserInterface/Controllers/CSSManager.js

```diff
@@ -389,6 +403,23 @@ WI.CSSManager = class CSSManager extends WI.Object
         return false;
     }
 
+    overrideUserPreference(preference, value)
+    {
+        for (let target of WI.targets) {
+            // COMPATIBILITY (macOS 13.0, iOS 16.0): `Page.overrideUserPreference()` did not exist yet.
+            if (target.hasCommand("Page.overrideUserPreference")) {
```

Comment:
NIT: Can we make this into a `continue` instead of having to indent everything?

Reply:
I'll leave this as-is for now. A follow-up patch implementing a backwards compatible way to override color-scheme/appearance will add another condition here to use the legacy protocol method. `continue` would miss it.